packages/modules-sdk/src/remote-query.ts

```diff
@@ -93,9 +93,11 @@ export class RemoteQuery {
     let fields: Set<string> = new Set()
     let relations: string[] = []
 
+    let hasStar = false
+
     for (const field of expand.fields ?? []) {
       if (field === "*") {
-        expand.fields = undefined
+        hasStar = true
```

Comment:
Let's name this something else, like `shouldSelectAll`

Reply:
nice one,m changed 👍